client/src/scenes/BootScene.ts

```diff
@@ -0,0 +1,53 @@
+import { Scene } from "phaser";
+import { HathoraClient } from "@hathora/client-sdk";
+import { RoomConnection } from "../connection";
+
+const client = new HathoraClient(process.env.APP_ID as string, process.env.COORDINATOR_HOST);
+
+class BootScene extends Scene {
```

Comment:
```suggestion
export class BootScene extends Scene {
```
and then you can get rid of the export at the end

Reply:
Sure thing, I'll update this.